### PR TITLE
Implement Must Have Media collection promo. Fixes #3063

### LINKED ISSUE #3063: New homepage carousel promo: Must-Have Media
Time to promote a new custom collection on the homepage. Similar to our recent “Games!” promotion, can we please get the following collection placed into the [Disco Pane management tool](https://addons.mozilla.org/en-US/firefox/discovery/modules) by **Tuesday, July 19** (ideally), but the following week works fine too. 

!! Reminder: please make the title in carousel image clickable to the collection’s landing page !! 

https://addons.mozilla.org/firefox/collections/mozilla/must-have-media/

It's fine to re-use the same background image we had on the Games! collection. 

Overlay text (title, blurb):

**Must-Have Media**
Take better screenshots, improve your online video experience, finally learn how to make a GIF, and other great media tools.

### PR DESCRIPTION
**Pull request by EnTeQuAk**
This is honestly based on https://github.com/mozilla/addons-server/commit/e0aaf06b42493b0f3d265dbe797de159ee5c0dd2 since I had no idea what else it needed so this is straight forward and it seems it works.

### CODE REVIEW

**Comment by EnTeQuAk** [comment]
This will need another l10n extract so that translators have at least a few minutes to translate it... or not.

File: src/olympia/legacy_discovery/modules.py [hunk 1]
```diff
@@ -264,3 +264,14 @@ class Games(CollectionPromo):
...
+
+class MustHaveMedia(CollectionPromo):
+    slug = 'Must-Have Media'
+    cls = 'promo promo-purple'
+    collection_author, collection_slug = 'mozilla', 'must-have-media'
+    title = _(u'Must-Have Media')
+    subtitle = _(u'Take better screenshots, improve your online video '
+                 u'experience, finally learn how to make a GIF, and other '
+                 u'great media tools.')
+    linkify_title = True
```
Comment (muffinresearch) [inline src/olympia/legacy_discovery/modules.py]: The other one didn't linkify the title - is this required?

**Comment by muffinresearch** [comment]
Tests?

**Comment by muffinresearch** [comment]
r+wc

File: src/olympia/legacy_discovery/modules.py (hunk 1 above)
Comment (EnTeQuAk) [inline src/olympia/legacy_discovery/modules.py]: It is a requirement according to the open issue.

**Comment by EnTeQuAk** [comment]
Yeah, well, I added some that test that we can use the collection and that it is linkified :laughing:

**Pull request merged**